f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml

```diff
@@ -449,6 +459,12 @@ Dow.prototype = {
           cds["need_for_entire_task_order_duration"] = ctb(cds.need_for_entire_task_order_duration);
           cds["traffic_per_domain_pair"] = JSON.parse(cds.traffic_per_domain_pair);
           if (!cds.need_for_entire_task_order_duration) {
+          // an error occurs when both the entire duration and selected periods is null
+          if (!cds.selected_periods) {
+            // default to entire duration when values are both null 
+            cds.need_for_entire_task_order_duration = true;
```

Comment:
This seems to be a reasonable default.  Was this behavior prescribed somewhere?  Just curious.

Reply:
@juliusfitzhugh-ccpo No, just made a judgement call. Ideally, if `CDS` is not required, the value is not used at all. Not sure, about the other place where the same logic is used (I think there is one other place in the script).

Reply:
Thank you